app/views/RoomView/List.js

```diff
@@ -74,12 +75,12 @@ class List extends React.Component {
 			}
 			this.messagesObservable = db.collections
 				.get('thread_messages')
-				.query(Q.where('rid', tmid))
+				.query(Q.where('rid', tmid), Q.or(Q.where('t', Q.notIn(hideSystemMessages)), Q.where('t', Q.eq(null))))
```

Comment:
Some `hideSystemMessages` don't follow message.types.
Like `mute_unmute` means `user-muted` and `user-unmuted`